packages/create-remix/index.ts

```diff
@@ -507,10 +507,24 @@ async function gitInitStep(ctx: Context) {
     while: async () => {
       let options = { cwd: ctx.cwd, stdio: "ignore" } as const;
       let commitMsg = "Initial commit from create-remix";
+      const commitUser = "create-remix";
+      const commitEmail = "<>"
```

Comment:
I think this should be the fallback and we should use the available global defaults first